zenodo/modules/deposit/templates/zenodo_deposit/delete.html

```diff
@@ -44,41 +48,33 @@ <h1>Delete record</h1>
         <p><h4>Are you ABSOLUTELY sure?</h4> This action <strong>CANNOT</strong> be undone! Deleting a record will
         permanently delete the associated files from disk.</p>
       </div>
+      {% if record_past_deletion_time %}
+        <div class="alert alert-warning" style="background-color:#df6d69" align="center"
+          <p> <h3 style="color:white"> ⚠️ This record is 1 month old! ⚠️ </h3> </p>
```

Comment:
```suggestion
          <p><h3> ⚠️ This record is 1 month old! ⚠️ </h3></p>
```

I would keep the default color. I think Bootstrap might be smart also and make this white text (since it's under an `alert-error` class)